Dockerfile

```diff
@@ -0,0 +1,57 @@
+# set base image (host OS)
+FROM python:3.9-slim-bullseye
+
+# set the working directory in the container
+WORKDIR /MixtComp
+
+# Checkout MixtComp
+COPY . .
```

Comment:
You should install all dependencies before copying MixtComp to fasten re-build of the image

Reply:
Thanks for the info